src/intrinsics/react/global.js

```diff
@@ -0,0 +1,74 @@
+/**
```

Comment:
It this only used for testing infra? Or for real implementation? It's not clear.
Should this be renamed to `src/intrinsics/react-mocks/global.js`?